src/pybind/mgr/ceph_module.pyi

```diff
@@ -67,7 +67,7 @@ class BaseMgrModule(object):
     def _ceph_lookup_release_name(self, release: int) -> str: ...
     def _ceph_cluster_log(self, channel: str, priority: int, message: str) -> None: ...
     def _ceph_get_context(self) -> object: ...
-    def _ceph_get(self, data_name: str) -> Dict[str, Any]: ...
+    def _ceph_get(self, data_name: str) -> Any: ...
```

Comment:
why? i think it is correct.

Reply:
`pg_ready` dumps a bool:

https://github.com/ceph/ceph/blob/5c9a3962b76374e2d94a7ef350a17dee8598b290/src/mgr/ActivePyModules.cc#L380-L383